readme.md

```diff
@@ -0,0 +1,104 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="logo.png" alt="logo" width="140"  height="123" />
+  <br/>
+</div>
+
+
+# 📗 Table of Contents
+
+- 📖 [About the Project](#about_project)
+  - [Screenshot](#screenshot)
+  - 🛠 [Built With](#built_with)
+- 💻 [Installation](#installation)
+- 👥[Author](#author)
+- 🤝 [Contributing](#contributing)
+- ⭐️ [Show your support](#support)
+- 🙏 [Acknowledgements](#acknowledgements)
+- 📝 [License](#license)
```

Comment:
- [ ] Excellent job utilizing the Microverse template for your README and personalizing it to represent your project! However, I noticed that some sections from the content table are missing. To make your README file even more comprehensive and meet the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), I kindly recommend adding all the required sections outlined in the [Microverse template](https://github.com/microverseinc/readme-template). Please note that the FAQ section is optional. Keep up the good work! 🎯
Your table of content should contain:
```
# 📗 Table of Contents

- [📖 About the Project](#about-project)
  - [🛠 Built With](#built-with)
    - [Tech Stack](#tech-stack)
    - [Key Features](#key-features)
  - [🚀 Live Demo](#live-demo)
- [💻 Getting Started](#getting-started)
  - [Setup](#setup)
  - [Prerequisites](#prerequisites)
  - [Install](#install)
  - [Usage](#usage)
  - [Run tests](#run-tests)
  - [Deployment](#deployment)
- [👥 Authors](#authors)
- [🔭 Future Features](#future-features)
- [🤝 Contributing](#contributing)
- [⭐️ Show your support](#support)
- [🙏 Acknowledgements](#acknowledgements)
- [❓ FAQ (OPTIONAL)](#faq)
- [📝 License](#license)
```